packages/web/src/components/token-value-slider/TokenValueSlider.module.css

```
@@ -10,13 +19,13 @@
   width: 576px;
 
   /* Text styles */
-  font-family: var(--font-family);
-  font-weight: var(--font-bold);
-  font-size: var(--font-s);
+  font-family: var(--harmony-font-family);
+  font-weight: var(--harmony-font-bold);
+  font-size: var(--harmony-font-s);
   line-height: 17px;
   letter-spacing: 1px;
   text-transform: uppercase;
-  color: #BEC5E0;
+  color: #bec5e0;
```

```suggestion
  color: var(--harmony-neutral)
```
(or `--harmony-n-800`)

---

yay